spec.bs

```diff
@@ -255,9 +267,12 @@ const publicKey = {
   authenticatorSelection: {
     userVerification: "required"
   },
+  
+  **TODO**: Extend definition (perhaps in future) to make this optional (so to only have userPresence: "required", and thereby support regions outside of europe) and remove friction of having to type in User PIN's?
```

Comment:
I think (though I'd need to double-check) that the spec doesn't currently require any particular `authenticatorSelection` options, though the user agent might (and Chrome currently does).

Reply:
So would Chrome then be deemed as compliant if it only supports a subset?
Is it your intention to support the other options as well in future; just not initially?

Reply:
> So would Chrome then be deemed as compliant if it only supports a subset?

Nope! :D

> Is it your intention to support the other options as well in future; just not initially?

I think this will go one of two ways: either we will hear no demand for other options and may restrict SPC for simplicity and/or clarity, or we will hear a compelling use-case and Chrome will add support for other options.

Reply:
I guess in that case then we'll need to pair back the spec at a later stage
@ianbjacobs : What is the best advice 
1.  have the spec adhere more to Chrome's initial implementation and note this as a future extension
2. have the spec be more generic and not have Chrome fully implement it?